wnfs/src/private/file.rs

```diff
@@ -875,7 +925,7 @@ impl PrivateForestContent {
         Ok(PrivateForestContent {
             key,
             base_name: forest.get_accumulated_name(&base_name),
-            block_count: block_index,
+            block_count: block_index as usize,
```

Comment:
`Can` you try changing `block_count: u64`? That may make stuff easier.

Reply:
Yeah I tried that, requires a bunch more changes to various types 🤔

Reply:
Ah okay. Then we'll leave it at that.